scripts/deploy_example_gcp

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPTS_DIR/common"
+
+source "$SCRIPTS_DIR/gcp_common"
+
+"$SCRIPTS_DIR/build_example" "hello_world"
+
+bazel_build_flags+=(
+    '--config=clang'
```

Comment:
Why is this needed? Perhaps add a comment.

Reply:
Turns out it is not needed. Removed.